routes/drafts.ts

```diff
@@ -0,0 +1,98 @@
+import express from 'express';
+import DraftsService from '../services/drafts';
+import { SchemaValidationError } from 'slonik';
+import { formatQueryErrorResponse } from '../helpers';
+import { PendingDraft, DraftUpdate } from '../types/draft';
+
+const router = express.Router();
+
+router.get("/", async (req, res) => {
+  try{
+    const drafts = await DraftsService.index();
+    return res.status(200).json(drafts);
+  }catch(e){
+    if (e instanceof SchemaValidationError) {
+      return res.status(400)
+        .json({message: 'Error fetching drafts: ' + formatQueryErrorResponse(e)})
+    }
+
+    console.log(e)
+    return res.status(500).json({message: 'Server Error'})
+  }
+});
+
+/*
```

Comment:
refactor: lets go ahead and start a test file. It's ok if you don't write any tests in it yet, but make a drafts.test.ts file in the tests folder and move this sample payload into it rather than leave it commented out in here.

Reply:
There was already a drafts.test.ts file. Should I just add the payload to the end and leave it commented out?

Reply:
yep that'll work!